content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +151,41 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The use of sysctl in pods can be controlled through `allowedUnsafeSysctls` and
+`forbiddenSysctls` fields on the PodSecurityPolicy.
 
-Sysctl annotation represents a whitelist of allowed safe and unsafe sysctls
-in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
-(which end in `*`). The string `*` matches all sysctls.
+By default, all safe sysctls are allowed. Currently, the whitelist of safe
+sysctls corresponds to the safe set mentioned above.
 
-Here is an example, it authorizes binding user creating pod with corresponding sysctls.
+Both `allowedUnsafeSysctls` and `forbiddenSysctls` are lists of plain sysctl names
+or sysctl patterns (which end in `*`). The string `*` matches all sysctls.
+
+The `allowedUnsafeSysctls` field excludes sysctls from the whitelist (`*` means
+no safe sysctls allowed).
+Any sysctl specified by the `forbiddenSysctls` is on the other hand allowed (`*`
+means all unsafe sysctls allowed).
+
+{{< note >}}
+**Note**: Even though the `allowedUnsafeSysctls` allows certain unsafe sysctls,
+if they are not allowed on the kubelet side, the pod fails to start.
```

Comment:
Should we mention the `--experimental-allowed-unsafe-sysctls` flag of kubelet here?

Reply:
Done.